### PR TITLE
Update Citadel in c.yaml

### PR DESCRIPTION
**Pull request by JosiahJack**
Update status to playable for Citadel and add note about needing original's data files for audio.

### CODE REVIEW

**Comment by JosiahJack** [comment]
Note that Github actions are struggling today, so may need reran once it's back up.  https://www.githubstatus.com/

File: games/c.yaml [hunk 1]
```diff
@@ -614,11 +615,11 @@
   content: free
```
Comment (cxong) [inline games/c.yaml:615]:
```suggestion
  content: commercial
```

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/c.yaml [hunk 2]
```diff
@@ -614,11 +615,11 @@
...
   originals:
   - System Shock
-  status: unplayable
+  status: playable
   repo: https://github.com/JosiahJack/Citadel
-  development: sporadic
+  development: active
   type: remake
-  updated: 2022-01-30
+  updated: 2023-04-15
```
Comment (cxong) [inline games/c.yaml:622]:
```suggestion
  updated: 2023-05-11
```

File: games/c.yaml (hunk 1 above)
Comment (JosiahJack) [reply games/c.yaml:615]: It is free though, unless this line refers to the original game?

Comment (cxong) [reply games/c.yaml:615]: this is referring to that the game requires original game assets to play - see https://github.com/opengaming/osgameclones/blob/master/schema/games.yaml#L59-L62

File: games/c.yaml (hunk 2 above)
Comment (cxong) [reply games/c.yaml]: Please note that this date is meant to reflect when the game was updated in OSGC, not the game project

File: games/c.yaml (hunk 1 above)
Comment (JosiahJack) [reply games/c.yaml]: Ah I missed that.  Updated.

**Review by cxong — APPROVED** [review]

**Review by cxong — APPROVED** [review]

**Comment by JosiahJack** [comment]
Did anything else need done here?  I don't have permissions to merge it.

**Pull request merged**